example/src/App.tsx

```diff
@@ -71,6 +73,13 @@ const App = () => {
         structServiceProvider={structServiceProvider}
         onInit={(ketcher: Ketcher) => {
           window.ketcher = ketcher;
+
+          KetcherLogger.settings = {
```

Comment:
My proposal is to remove this from here. Actually, for 99% of users logger won't be required. It is needed for cases, when we will directly ask customer like "Open the console, write this command, and send logs to us". 
So, can you additionally add to description, how logger can be enabled via Dev Tools console?

Reply:
To enable logger type in the console:
`window.ketcher.logger.enabler = true`

The default log level is "ERROR" or 0. 
To change log level type in the console:
`window.ketcher.logger.level = 1` to enable warnings
2 to enable infos
3 to enable logs

To show traces on logs and infos, type in the console:
`window.ketcher.logger.showTrace = true`

Reply:
Done